tennis.py

```diff
@@ -15,8 +15,11 @@ def __init__(self, player1Name, player2Name):
     def won_point(self, playerName):
         if playerName == self.player1.name:
             self.player1.points += 1
-        else:
+        if playerName == self.player2.name:
```

Comment:
```suggestion
        elif playerName == self.player2.name:
```
Attention, sinon tu vas "rentrer" dans le `else` si playerName == player1.name